v3/tutorials/09-cumulus/d-rococo/index.mdx

```diff
@@ -25,162 +25,127 @@ keywords:
+The goal of this more dynamic allocations aims at helping teams to test their runtimes more often, and in a more streamlined manner.
+At this point, requesting a permanent or temporary parachain slot still a privileged (sudo) operation to be performed by the test chain's operator (using a call to the `assignedSlots` pallet's dispatchable functions) but the long-term plan is to make it a community-driven process via Rococo's governance framework.
 
-#### Bidding
+Follow these steps to request a temporary slot:
 
-Anyone with a fully onboarded parathread can make a bid to win a parachain slot for their para ID.
-They need to out-bid all others participating in the slot auction.
+1. In the [Polkadot-JS Apps for Rococo](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frococo-rpc.polkadot.io#), go to `Developer -> Extrinsics`.
+1. Select the account with enough ROC balance that you wish to use to submit a transaction with.
+1. Select the `assignedSlots` pallet.
+1. Choose the `assignTempParachainSlot` function. 
+1. Insert your reserved para ID. Make sure this matches the one you reserved from the previous section!
+1. Select `Current` for the `LeasePeriodStart`. If the current slot is full, you will be assigned the next available slot.
+1. Sign and submit the transaction.
 
-You can do so in Polkadot-JS App [Network -> Parachains -> Auctions](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frococo-rpc.polkadot.io#/parachains/auctions) page.
+> The Rococo `sudo` key is controlled by Parity tech at this time, so to have the operation required to get a slot, please on the [Parachain Technical matrix channel](https://matrix.to/#/#parachain-technical:matrix.parity.io) once you complete the above and are ready to connect!
+> They will initiate the slot for you ASAP.
 
-Pick your para ID, how much you want to bid, and the slots you want to bid for:
+Given a period lease duration of 1 day, the current settings for assigned parachain slots on Rococo are (at the time of writing):
+- **Permanent slot least duration**: 1 year (365 days)
+- **Temporary slot least duration**: 3 days
+- **Maximum number of permanent slots**: up to 25 permanent slots
+- **Maximum number of temporary slots**: up to 20 temporary slots
+- **Maximum temporary slots allocated per leased period**: up to 5 temporary slots per 3-day temporary lease periods
 
-![parachain-bid.png](../../../../src/images/tutorials/09-cumulus/parachain-bid.png)
+These are subject to change, based on the needs of the community.
 
-### Crowdloans
+🎉**_Congratulations!_**🎉  
+With your slot activated by the Parity team, you're now able to test your parachain on the Rococo test net!
+Please do note that when your temporary slot's lease ends, the parachain is automatically downgraded to a parathread. Registered and approved slots are cycled through automatically in a round-robin fashion, so you will expect to come back online as a parachain from time to time.
```

Comment:
```suggestion
Please do note that when your temporary slot's lease ends, the parachain is automatically downgraded to a parathread. 
Registered and approved slots are cycled through automatically in a round-robin fashion, so you will expect to come back online as a parachain from time to time.
```
I thought that once your temporary slot is up and you're downgraded to a parathread, you need to re-register for a temporary slot again. ".. you will expect to come back online as a parachain from time to time" makes it sound like you could just be reassigned a slot automatically even after your slot period runs out. Is this the case?

Reply:
cc @NukeManDan.

Reply:
@NukeManDan @sacha-l so, when a para temp slot's lease ends, the parachain is downgraded back to a parathread (this is done by the `slots` palet's `manage_lease_period_start` function). A new batch of parathreads are then upgraded to parachains for the duration of a temp slot lease (this is done by the `assigned_slots` pallet's `allocate_temporary_slot_leases` function). The catch is that a temp slot does not expire (once registered, it remains registered until one calls `unassign_parachain_slot`), but it has multiple slot leases (in the `slots` pallet's terminology) that have a fixed duration (defined by the `TemporarySlotLeasePeriodLength` param). Does that make sense ? 😅

Reply:
Makes sense! OK. That would imply that the more registered paraID's, the least likely a newly added paraID will be chosen for the next available slot? Is there a cap to the number of para IDs that can be registered?